lib/rules/at-rule-semicolon-space-before/README.md

```diff
@@ -0,0 +1,45 @@
+# at-rule-semicolon-space-before
+
+Require a single space or disallow whitespace before the semicolons of regular (non-nested) at-rules.
+
+```css
+@import "components/buttons";
+/**                         ↑
+ * The space before this semicolon */
+```
+
+## Options
+
+`string`: `"always"|"never"`
+
+### `"always"`
+
+There *must always* be a single space before the semicolons.
+
+The following patterns is considered a warning:
+
+```css
+@import "components/buttons";
+```
+
+The following pattern is *not* considered a warning:
```

Comment:
I'll update the [valid and invalid example highlighting](https://github.com/stylelint/stylelint.io/blob/master/plugins/remark-stylelint-pattern-validity/index.js)) on the website to accept these singular examples.